src/services/authentication/index.js

```diff
@@ -8,17 +8,23 @@ const { hooks } = require('./hooks');
 const { authConfig } = require('./configuration');
 
 class SCAuthenticationService extends AuthenticationService {
+	async getUserData(user, account) {
+		return {
+			accountId: account._id,
+			systemId: account.systemId,
+			userId: user._id,
+			schoolId: user.schoolId,
+			roles: user.roles,
+		};
+	}
 	async getPayload(authResult, params) {
 		let payload = await super.getPayload(authResult, params);
+		const userData = await this.getUserData(authResult.account.userId, authResult.account._id);
```

Comment:
here we need to load the user first using the userId?
```suggestion
			const user = await this.app.service('usersModel').get(authResult.account.userId);
		const userData = await this.getUserData(user, authResult.account._id);
```